src-tauri/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+// Learn more about Tauri commands at https://tauri.app/v1/guides/features/command
```

Comment:
```suggestion
// Learn more about Tauri commands at https://tauri.app/develop/calling-rust
```

We should also change the comment in `interop.js`